component/rbac.libsonnet

```diff
@@ -31,9 +31,8 @@ local sudoGroupSubjects = std.map(
 local sudoClusterRole = kube.ClusterRole('sudo-impersonator') {
   rules: [ {
     apiGroups: [ '' ],
-    resources: [ 'users' ],
+    resources: [ 'users', 'serviceaccounts' ],
```

Comment:
While we're at it, shouldn't we also grant impersonation permissions on `groups` for `kubectl --as-group=<groupname>`?

Reply:
I've added groups and expanded the docs.